site/docs/v1/tech/samlv2/tableau-cloud.adoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: doc
+title: SAML v2 & Tableau Cloud
+description: Setting up Tableau Cloud to login using FusionAuth via SAML v2
+navcategory: login-methods
+---
+
+== Overview
+
+link:https://www.tableau.com/[Tableau] is one of the most advanced data visualization tools used for business intelligence and data analysis. This document covers the configuration necessary to get Tableau Cloud working with FusionAuth as the identity provider using SAML v2, allowing users to log in to your Tableau Cloud account using their FusionAuth credentials.
+
+=== Prerequisites
+
+This document assumes you have a running instance of FusionAuth and a working Tableau Cloud application. You will also need admin accounts for both to configure them correctly.
+
+Finally, you'll need a FusionAuth user that you will use to sign into Zendesk. You can use an existing user or create a new user for this purpose.
+
+=== Instructions
+
+We also recommend taking a look at link:https://help.tableau.com/current/online/en-us/saml_config_site.htm[Tableau Cloud SSO docs], but here are the instructions.
+
+In your Tableau Cloud dashboard, navigate to [breadcrumb]#Settings -> Authentication#. Select [uielement]#Enable an additional authentication method#, choose [uielement]#SAML# and click [uielement]#Edit connection# to open the settings.
+
+image::samlv2/tableau/navigation.png[Steps to enable SAML in Tableau CLoud.,width=700,role=box-shadow bottom-cropped top-cropped]
+
+Copy both [field]#Tableau Cloud entity ID# and [field]#Assertion Consumer Service URL (ACS)# from [uielement]#1. Export metadata from Tableau Cloud# and write them down. You'll use them shortly.
```

Comment:
```suggestion
Copy both [field]#Tableau Cloud entity ID# and [field]#Assertion Consumer Service URL (ACS)# from [uielement]#Export metadata from Tableau Cloud# and write them down. You'll use them shortly.
```

Reply:
Let's get rid of all the numbers from the tableau navigation. I understand why you had them, but it looks weird.